src/app/pages/PerpetualPage/utils/bscWebsocket.ts

```diff
@@ -0,0 +1,46 @@
+import Web3 from 'web3';
+import PerpetualManager from '../components/TradingChart/IPerpetualManager.json';
```

Comment:
Use src/utils/blockchain/abi/PerpetualManager.json instead remove the added ABI from the TradingChart directory.